include/boost/geometry/algorithms/detail/sections/sectionalize.hpp

```diff
@@ -935,12 +942,16 @@ inline void sectionalize(Geometry const& geometry,
     ring_identifier ring_id;
     ring_id.source_index = source_index;
 
+    using box_type = typename Sections::box_type;
+    using point_type = typename geometry::point_type<box_type>::type;
+
     dispatch::sectionalize
         <
             typename tag<Geometry>::type,
             Geometry,
             Reverse,
-            DimensionVector
+            DimensionVector,
+            point_type
         >::apply(geometry, robust_policy, sections,
                  strategy,
                  ring_id, max_count);
```

Comment:
If `sections` are passed as function argument then you don't have to pass `point_type` as template parameter. `box_type` should rather be taken from the type of `sections` inside `apply()`. The number of template parameters should be minimal that is required for the dispatching to work. `point_type` is not needed for that. It's an additional information needed only in `apply()`.

Reply:
Yes, it’s true. I started in another way, which is why I added them.
But you are right, I can remove them now. I’ll do that. Thanks.